apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -124,7 +125,7 @@ function admin_body_classes( $classes ) {
 		$classes .= ' slider-width-workaround';
 	}
 
-	if ( use_font_smooth_antialiased() ) {
+	if ( use_font_smooth_antialiased() && ! is_network_admin() ) {
```

Comment:
TIL network admin is a core feature of WP multi-sites, not a wpcom only thing. That means `is_network_admin` will be available on atomic.